schemasheets/examine_schemasheet.py

```diff
@@ -0,0 +1,7 @@
+from schemasheets.schemasheet_datamodel import SchemaSheet
+
+specification = "../populated_with_generated_spec_subset.tsv"
```

Comment:
avoid relative links. See e.g. https://stackoverflow.com/questions/2817264/how-to-get-the-parent-dir-location

Reply:
thanks for the reminder. solving by deleting the script.